understanding/22/consistent-help.html

```diff
@@ -71,7 +71,7 @@ <h2>Intent of Consistent Help</h2>
 
         <p>The human contact mechanism enables a person to express what they are looking for using their own words. For some with cognitive disabilities, this may be the best way for them to find an answer to their problem.</p>
 
-        <p>For pages for which no human support is available, it is best if a self-help option says that no human support is available.  Self-help options should go beyond allowing the user to search within the site. Contextual help is still recommended (see <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-context-help.html">3.3.5 Help</a> for more information), but a self-help option should provide a single location that makes it easier for people with cognitive disabilities to understand what help is available without having to hunt for it. While some people may easily be able to identify that no support would be available for a particular type of Web site, this may not be apparent to some users with disabilities.</p>
+        <p>For pages for which no human support is available, it is best if a self-help option says that no human support is available.  Self-help options should go beyond allowing the user to search within the site. Contextual help is still recommended (see <a href="../20/help.html">3.3.5 Help</a> for more information), but a self-help option should provide a single location that makes it easier for people with cognitive disabilities to understand what help is available without having to hunt for it. While some people may easily be able to identify that no support would be available for a particular type of Web site, this may not be apparent to some users with disabilities.</p>
```

Comment:
is this change not still linking to the 2.0 understanding? or is this magically changed on publication to be the 3.3.5 understanding under WCAG 2.2?

Reply:
That's fine, the publishing process adapts that. (Anything with an absolute address to https://blahblah/WCAG20/ does need changing.)